test/e2e/predictor/test_paddle.py

```diff
@@ -119,3 +119,48 @@ def test_paddle_runtime():
     assert np.argmax(res["predictions"][0]) == 17
 
     kserve_client.delete(service_name, KSERVE_TEST_NAMESPACE)
+
+
+@pytest.mark.slow
+def test_paddle_v2_kserve():
+    predictor = V1beta1PredictorSpec(
+        min_replicas=1,
+        model=V1beta1ModelSpec(
+            model_format=V1beta1ModelFormat(
+                name="paddle",
+            ),
+            runtime="kserve-paddleserver",
```

Comment:
I think we need the protocol: v2 in spec?

Reply:
Created https://github.com/kserve/kserve/issues/2784 to track this issue, if we specify v2 then kserve currently selects the mlserver runtime